src/iris/api.py

```diff
@@ -700,6 +708,26 @@ def process_resource(self, req, resp, resource, params):  # pragma: no cover
         if resource.allow_read_no_auth and method == 'GET':
             return
 
+        # Ignore HMAC requirements for custom webhooks
+        if req.env['PATH_INFO'].startswith('/v0/webhooks/'):
+            app_name = req.get_param('application', required=True)
+            app = cache.applications.get(app_name)
+            if not app:
+                logger.warn('Tried authenticating with nonexistent app: "%s"', app_name)
+                raise HTTPUnauthorized('Authentication failure', '', [])
+
+            req.context['app'] = app
+
+            # determine if we're correctly using an application key
+            api_key = req.get_param('key', required=True)
+            if not api_key:
```

Comment:
When `required=True`, Falcon will raise a 4xx if the param is not defined. You should be able to remove the `if not api_key` check